.gitignore

```diff
@@ -0,0 +1,3 @@
+.vscode
+.editorconfig
```

Comment:
Are you abandoning #591 that adds `.editorconfig`? AFAICT, #591 ball is on your side; it may be better to advance that PR before this one. I do not insist on removing this entry from this PR, just pointing out a conflict between your open PRs.

Reply:
No, I am not, this is a side-step to have one for myself, while waiting for the global one to be accepted (or not).
I''ve submitted PR#660, cleaning up PERL code, which can be a step forward for PR#591